src/App.js

```diff
@@ -15,7 +15,7 @@ import Button from 'react-bootstrap/Button'
 import Modal from 'react-bootstrap/Modal'
 import NavDropdown from 'react-bootstrap/NavDropdown'
 import { useDispatch, useSelector } from 'react-redux'
-import ProductPage from './components/product-list-component/ProductPage'
+import { ProductDescription} from './components/product-list-component/Product__Description/ProductDescription'
```

Comment:
name of the folder should be product-description 
and 
and the component file name should be product-description.component.jsx  not ProductDescription.js
same goes to css file